ql/src/test/results/clientpositive/llap/jdbc_handler.q.out

```diff
@@ -118,6 +118,25 @@ POSTHOOK: type: QUERY
 POSTHOOK: Input: default@ext_simple_derby_table
 #### A masked pattern was here ####
 200
+PREHOOK: query: insert into ext_simple_derby_table values(100)
+PREHOOK: type: QUERY
+PREHOOK: Input: _dummy_database@_dummy_table
+PREHOOK: Output: default@ext_simple_derby_table
+POSTHOOK: query: insert into ext_simple_derby_table values(100)
+POSTHOOK: type: QUERY
+POSTHOOK: Input: _dummy_database@_dummy_table
+POSTHOOK: Output: default@ext_simple_derby_table
+PREHOOK: query: select * from ext_simple_derby_table
+PREHOOK: type: QUERY
+PREHOOK: Input: default@ext_simple_derby_table
+#### A masked pattern was here ####
+POSTHOOK: query: select * from ext_simple_derby_table
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@ext_simple_derby_table
+#### A masked pattern was here ####
+20
+200
+100
 PREHOOK: query: CREATE EXTERNAL TABLE tables
```

Comment:
Can we add additional testing? For instance:
1) Inserting rows with column values using all different Hive primitive types (boolean, tinyint, smallint, int, bigint, float, double, decimal, char, varchar, string, date, timestamp, etc.)? If any of those types are not supported, e.g., complex types, we already throw a user friendly error but please add a negative tests for those. 
2) Inserting multiple rows.
3) CTAS statement.

Reply:
> Can we add additional testing? For instance:
> 
> 1. Inserting rows with column values using all different Hive primitive types (boolean, tinyint, smallint, int, bigint, float, double, decimal, char, varchar, string, date, timestamp, etc.)? If any of those types are not supported, e.g., complex types, we already throw a user friendly error but please add a negative tests for those.
> 2. Inserting multiple rows.
> 3. CTAS statement.

Added more test cases!